cg_hermes/constants/tags.py

```diff
@@ -574,6 +598,35 @@ def description(self) -> str:
         return descriptions.get(self, "Description not available")
 
 
+class RarediseaseTags(StrEnum):
+    ANNOTATION: str = "annotation"
+    EKLIPSE_DEL: str = "eklipse-del"
+    EKLIPSE_GENES: str = "eklipse-gene"
+    EKLIPSE_PNG: str = "eklipse-png"
+    HAPLOGREP: str = "haplogrep"
+    NGSBITS: str = "ngsbits_samplegender"
```

Comment:
I think we're trying to use sex instead of gender, so maybe we should do it with the tags as well
```suggestion
    NGSBITS: str = "ngsbits-sample-sex"
```

Reply:
samplegender is the name of the ngsbits tool unfortunately

Reply:
I see 👍